From: Suresh Reddy <suresh.reddy@broadcom.com>
Date: Sun, 9 Oct 2016 09:58:53 +0530
Subject: be2net: Enable VF link state setting for BE3
Patch-mainline: v4.9-rc1
Git-commit: dc6e8511ff7141141578bac559565c55a1e14ad8
References: bsc#1021121 FATE#321675

The VF link state setting feature now works on BE3 chips too from
FW ver 11.1.192.0 onwards.

Signed-off-by: Suresh Reddy <suresh.reddy@broadcom.com>
Signed-off-by: Sriharsha Basavapatna <sriharsha.basavapatna@broadcom.com>
Signed-off-by: David S. Miller <davem@davemloft.net>
Acked-by: David Chang <dchang@suse.com>
---
 drivers/net/ethernet/emulex/benet/be_cmds.c |    2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

--- a/drivers/net/ethernet/emulex/benet/be_cmds.c
+++ b/drivers/net/ethernet/emulex/benet/be_cmds.c
@@ -4977,7 +4977,7 @@ int be_cmd_set_logical_link_config(struc
 {
 	int status;
 
-	if (BEx_chip(adapter))
+	if (BE2_chip(adapter))
 		return -EOPNOTSUPP;
 
 	status = __be_cmd_set_logical_link_config(adapter, link_state,
